backend/src/main/java/com/benine/backend/http/PresetCreationHandler.java

```diff
@@ -40,20 +48,43 @@ public void handle(HttpExchange exchange) throws IOException {
         IPCamera ipCamera = (IPCamera)camera;
         
         Preset preset = createPreset(ipCamera);
+        int presetID = preset.getId();
         
         //Adding the new preset to the database
         getCameraController().addPreset(cameraID, preset);
+        
+        //Create corresponding image
+        createImage(preset, cameraID, presetID);
         responseSuccess(exchange);
       
       }
-    } catch (SQLException e) {
-      System.out.println(e.toString());
+    } catch (SQLException | StreamNotAvailableException e) {
       getLogger().log("Preset can not be added to the database", LogEvent.Type.CRITICAL);
-    }
+    } 
 
   }
   
   /**
+   * Create an image that belongs to a preset
+   * @param cameraID the ID of the camera used with the preset
+   * @throws StreamNotAvailableException exception if there's no stream for the camera available
+   * @throws IOException exception thrown if the input is wrong. 
+   */
+  public static void createImage(Preset preset, int cameraID, int presetID) throws StreamNotAvailableException, IOException {
+    ServerController serverController = ServerController.getInstance();
+    StreamReader streamReader = serverController.getStreamController().getStreamReader(cameraID);
+    BufferedImage bufferedImage = streamReader.getSnapShot(); 
+    
+    //Rescale image so it loads faster.
+    //BufferedImage buffer = (BufferedImage)bufferedImage.getScaledInstance(360, 235, BufferedImage.SCALE_DEFAULT);
+    
+    File path = new File("static" + File.separator + "presets" + File.separator + cameraID + "_" + presetID + ".jpg");
```

Comment:
I think the path should be File.separator + "static" + File.separator + "presetImages" + File.separator + cameraID + "_" + presetID + ".jpg"